swagger/api/v1/b0_pe_threads.yaml

```diff
@@ -0,0 +1,16 @@
+post:
+  operationId: createThread
```

Comment:
Can we get only 200, 201, 400 HTTP status? Couldn't we get a 404 or 500 by any chance?

Reply:
Yes, we could add them. But we do not intend to provide any special behavior e.g. we are not trying to send back a custom message for 500. It will be a standard message with a generic implementation for 500 etc.
I suggest to not add these statuses in specs. But these should indeed be in the implementation.